schemas/2019-01-21/Microsoft.Kusto.json

```diff
@@ -0,0 +1,402 @@
+﻿{
+  "id": "https://schema.management.azure.com/schemas/2019-01-21/Microsoft.Kusto.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Kusto",
+  "description": "Microsoft Kusto Resource Types",
+  "resourceDefinitions": {
+    "clusters": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The Cluster name"
+        },
+        "type": {
+          "enum": [
+            "Microsoft.Kusto/clusters"

```

Comment:
>clusters [](start = 29, length = 8)

do you need to add them to the root schemas?

Reply:
Oops thank you, fixing